dts/bindings/dac/gd,gd32-dac.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 BrainCo Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: GigaDevice GD32 series DAC module
+
+compatible: "gd,gd32-dac"
+
+include: [dac-controller.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    rcu-periph-clock:
+      type: int
+      description: Reset Control Unit Peripheral Clock ID
+      required: true
+
+    num-channels:
+      type: int
+      description: GD32 DAC output channels number
+      required: true
+
+    reset-val:
+      type: int
+      description: GD32 DAC output reset value
+      required: true
```

Comment:
suggestion: you could default to the SoC default value here (dropping requirement)

Reply:
done.